tests/incremental/13-restart-write/09-mutex-self-fix.json

```diff
@@ -0,0 +1,9 @@
+{
+  "incremental": {
+    "restart": {
+      "sided": {
+        "enabled": false
+      }
```

Comment:
I think it would be good to include the `incremental.restart.write-only` property in this configuration even though it is set to true by default, because it is essential for this test and these configurations break easily when we make changes to our default configurations.